app/assets/v2/js/pages/bounty_details.js

```diff
@@ -20,10 +20,16 @@ var gitcoin_ize = function(key, val) {
   if (!_truthy(val)) {
     return [ null, null ];
   }
-  return [ key, '<a href="https://gitcoin.co/profile/' + val + '" target="_blank" rel="noopener noreferrer">@' + val.replace('@', '') + '</a>' ];
+  return [ key, '<a href="https://gitcoin.co/profile/' + val + '" target="_blank" rel="noopener noreferrer">' + val.replace('@', '') + '</a>' ];
 };
 
 var email_ize = function(key, val) {
+
+  if (val == 'Anonymous' || val == '') {
+    $('#bounty_owner_email').hide();
```

Comment:
Maybe is better to just .remove() the elements since we don't want to show it.

Reply:
derp derp ! why didn't I think of that 😂